packages/veritone-react-common/src/components/Video/RestartMediaButton/index.js

```diff
@@ -0,0 +1,39 @@
+import { string, func, shape } from 'prop-types';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  actions: shape({
+    seek: func
+  }),
+  className: string
+};
+
+export default class RestartMediaButton extends Component {
+  constructor(props, context) {
```

Comment:
Please follow the structure of our other components, we're not using constructor and PropTypes should be specified inside of the component.